### ISSUE TITLE
Crash when running rebuilt CIA

### ISSUE THREAD

**Issue by mariomadproductions** [Asia81/HackingToolkit9DS#17]
I'm using V11. When I run a rebuilt _"Super Mario Land (USA) (v16) - built with make_cdn_cia using ticket from CDN-FX (md5: a9c4ec2cfe117620b08d8ec68b54963c)"_ , with no changes made, I get a Luma3DS crash.

**Comment by Asia81** [comment]
Yes, a known error, still working on a fix for the cia rebuild script.
